draft-omara-sframe.md

```diff
@@ -458,23 +458,19 @@ o [Optional] A signature algorithm
 This document defines the following ciphersuites:
 
 
-| Value  | Name                           | Nh | Nk | Nn | Reference |
-|:-------|:-------------------------------|:---|:---|:---|:----------|
-| 0x0001 | AES\_CM\_128\_HMAC\_SHA256\_8  | 32 | 16 | 12 | RFC XXXX  |
-| 0x0002 | AES\_CM\_128\_HMAC\_SHA256\_4  | 32 | 16 | 12 | RFC XXXX  |
-| 0x0003 | AES\_GCM\_128\_SHA256          | 32 | 16 | 12 | RFC XXXX  |
-| 0x0004 | AES\_GCM\_256\_SHA512          | 64 | 32 | 12 | RFC XXXX  |
+| Value  | Name                           | Nh | Nk | Nn |Auth Tag Len | Reference |
+|:-------|:-------------------------------|:---|:---|:---|:------------|:----------|
+| 0x0001 | AES\_CM\_128\_HMAC\_SHA256\_64 | 32 | 16 | 12 |          64 | RFC XXXX  |
+| 0x0002 | AES\_CM\_128\_HMAC\_SHA256\_32 | 32 | 16 | 12 |          32 | RFC XXXX  |
+| 0x0003 | AES\_GCM\_128\_SHA256          | 32 | 16 | 12 |         128 | RFC XXXX  |
+| 0x0004 | AES\_GCM\_256\_SHA512          | 64 | 32 | 12 |         256 | RFC XXXX  |
```

Comment:
The auth tag here is still 128 bits.

Reply:
The rest of values (Nh/NK/Nn) are in bytes, right?

Reply:
Correct.  So either the auth tag length should be in bytes, or we should clarify that it's in bits.  (We could also assign it a cute name like the other columns, say `Nt`.)  In any case, AES-128-GCM and AES-256-GCM both have a 16-byte / 128-bit tag.

Reply:
```suggestion
| 0x0004 | AES\_GCM\_256\_SHA512          | 64 | 32 | 12 |         128 | RFC XXXX  |
```